tests/test_api_validate.py

```diff
@@ -747,22 +747,6 @@ def test_05_check_serial_with_no_user(self):
             self.assertEqual(result.get("status"), True)
             self.assertEqual(result.get("value"), True)
 
-    def test_05_check_serial_with_no_user(self):
```

Comment:
Why did you remove this test?

Reply:
Ah, the github diff doesn't show but this function is a duplicate. The same function name with the same code is used just before this one :-)

Reply:
thanks.